content/pages/coc.rst

```diff
@@ -0,0 +1,88 @@
+Code of Conduct
+===============
+
+Pyninsula is dedicated to providing a harassment-free conference experience
+for everyone, regardless of gender, gender identity and expression,
+sexual orientation, disability, physical appearance, body size, race,
+age or religion.
+We do not tolerate harassment of conference participants in any form.
+Sexual language and imagery is not appropriate in the meetup,
+including talks.
+Pyninsula participants violating these rules may be sanctioned or
+expelled from the venue at the discretion of the meetup organizers.
+
+Harassment includes, but is not limited to:
+
+* Offensive verbal comments related to gender, gender identity and expression, sexual orientation, disability, physical appearance, body size, race, age, or religion.
+* Sexual images in public spaces
+* Deliberate intimidation, stalking, or following 
+* Harassing photography or recording
+* Sustained disruption of talks or other events
+* Inappropriate physical contact
+* Unwelcome sexual attention
+* Advocating for, or encouraging, any of the above behaviour
+
+Enforcement
```

Comment:
> This are what we're "allowed to do", not "how to do it" -- these are rules, not policies. Optimizing policies to avoid retaliation is beyond the scope of this PR.

i take your point, and indeed part of my concern is the effectiveness of rules without policies.  but that's out of scope for this pr, and now there's https://github.com/moshez/pyninsula/wiki/Behavior-policy

at the same time, this _does_ include a policy _for reporters_, which necessarily stops after they make their report.  i'd like to strengthen this:

> You won't be asked to confront anyone and we won't tell anyone who you are.

so that it's clear that both are true in subsequent meetups, too.